config/locales/nl.yml

```diff
@@ -84,6 +84,7 @@ nl:
       user: Gebruiker
       category: Categorie
       billed: Gefactureerd
+      unbill: Herzie facturatie
```

Comment:
not particularly strong as it is an actual undo. Would prefer 'Facturatie ongedaan maken' of 'Facturatie terugdraaien'